test/cxx_api_tests/RoadRunnerAPITestsWithLLJit.cpp

```diff
@@ -0,0 +1,859 @@
+//
+// Created by Ciaran on 07/05/2021.
+//
+
+#include "RoadRunnerAPITests.h"
+#include "rrConfig.h"
+
+using namespace rr;
+
+class RoadRunnerAPITestsWithLLJit : public RoadRunnerAPITests {
+public:
+    RoadRunnerAPITestsWithLLJit() 
+        : RoadRunnerAPITests(){
+        Logger::setLevel(Logger::LOG_ERROR);
+        Config::setValue(Config::LLVM_BACKEND, Config::LLVM_BACKEND_VALUES::LLJIT);
+    }
+};
+
+
+
+TEST_F(RoadRunnerAPITestsWithLLJit, DefaultJacobianMode){
+    DefaultJacobianMode();
```

Comment:
This test and the others in this file look like they are prototypes and not full tests yet.  True?

Reply:
Nope, figured it out--you renamed the other test .h so that you could subclass these tests with different options.  Got it!